app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != 0:
```

Comment:
The condition `parts[0] != 0` is incorrect. `parts[0]` is expected to be a string, not an integer. If you want to check if the first part is a specific command, you should compare it to a string, like `parts[0] != 'copy'`.